coffea/processor/executor.py

```diff
@@ -1027,33 +800,13 @@ def run_uproot_job(
             treename can also be defined in fileset, which will override the passed treename
         processor_instance : ProcessorABC
             An instance of a class deriving from ProcessorABC
-        executor : callable
-            A function that takes 3 arguments: items, function, accumulator
+        executor : ExecutorBase instance
```

Comment:
fileset, treename, and processor_instance should be removed from the doc.

Reply:
Good catch. The executor instance still is a Callable, which implements `__call__` with the same signature as before. So the doc string is still valid, however I would extend it to include that it needs to derive from `ExecutorBase`. I hope this is fine with you :)